contracts/BaseStrategy.sol

```diff
@@ -869,9 +487,10 @@ abstract contract BaseStrategyInitializable is BaseStrategy {
         address _vault,
         address _strategist,
         address _rewards,
-        address _keeper
+        address _keeper,
+        uint256 _changeLimitRatio
```

Comment:
lets add a sensible default and remove it from initialize, we can add a setter to update later